src/pages/steps/CalendarStep/CalendarOptionToggle.tsx

```diff
@@ -38,17 +46,49 @@ export const CalendarOptionToggle = ({
         minHeight={parseSpacing(7)}
         flex={1}
       />
-      <ToggleBox
-        onClick={onChooseFixedDays}
-        active={isFixedDays}
-        icon={
-          <CustomIcon name={CustomIconVariants.CALENDAR_MULTIPLE} width="80" />
-        }
-        text={t('create.calendar.types.fixed_days')}
-        minHeight={parseSpacing(7)}
-        flex={1}
-        disabled={disableChooseFixedDays}
-      />
+      {!(isCultuurkuurFeatureFlagEnabled && isCultuurkuurEvent) && (
+        <ToggleBox
+          onClick={onChooseFixedDays}
+          active={isFixedDays}
+          icon={
+            <CustomIcon
+              name={CustomIconVariants.CALENDAR_MULTIPLE}
+              width="80"
+            />
+          }
+          text={t('create.calendar.types.fixed_days')}
+          minHeight={parseSpacing(7)}
+          flex={1}
+          disabled={disableChooseFixedDays}
+        />
+      )}
+      {isCultuurkuurFeatureFlagEnabled && isCultuurkuurEvent && (
+        <ToggleBox
+          onClick={onChooseFixedDays}
+          active={isFixedDays}
+          icon={
+            <CustomIcon
+              name={CustomIconVariants.CULTUURKUUR_CALENDAR}
+              width="80"
+            />
+          }
+          text={
+            <Inline spacing={2}>
+              <Box>
+                <p>{t('create.calendar.types.cultuurkuur')}</p>
+              </Box>
+              <Tooltip
+                content={t('create.calendar.types.cultuurkuur_tip')}
+                id={t('create.calendar.types.cultuurkuur_tip')}
+                placement="bottom"
+              />
+            </Inline>
+          }
+          minHeight={parseSpacing(7)}
+          flex={1}
+          disabled={disableChooseFixedDays}
+        />
+      )}
```

Comment:
Since the two conditions are 90% the same I would maybe instead always render the `ToggleBox`, and pass the identical props to it. But then do like `const props = isCultuurBlabla ? {icon: a, text: a} : {icon: b, text: b}` and pass that to it